plugins/feeds/public/fumik0.py

```diff
@@ -0,0 +1,83 @@
+import requests
+from datetime import datetime, timedelta
+import logging
+
+from core.observables import Url, Ip, Hash
+from core.feed import Feed
+from core.errors import ObservableValidationError
+from core.config.config import yeti_config
+
+
+class Fumik0Tracker(Feed):
+
+    default_values = {
+        "frequency": timedelta(hours=1),
+        "name": "Fumik0Tracker",
+        "source": "https://tracker.fumik0.com/api/get-samples",
+        "description": "This feed contains md5/sha1/sha256/urls",
+    }
+
+    def update(self):
+        resp = requests.get(self.source, proxies=yeti_config.proxy)
+        if resp.ok:
+            for block in resp.json():
+                self.analyze(block)
+
+    def analyze(self, block):
+
+        """
+        block example
+           {u"first_seen": u"2019-03-13 14:38:12",
+             u"hash": {u"md5": u"e1513c048e520e8d5fc5999d82994ea7",
+                       u"sha1": u"f09f66c3bd4cd54cc030b7d102be32376fd993b5",
+                       u"sha256": u"bbb450f1f68735054af6d1c64bd3a7e62f9977d40eeb286340d8bc1dac6f7e7e"},
+             u"hash_seen": 1,
+             u"id": u"5c8915d47a324f51d460e8e5",
+             u"sample": {u"name": u"vdvdv.exe", u"size": u"600576"},
+             u"server": {u"AS": u"AS197695",
+                         u"country": u"ru",
+                         u"ip": u"37.140.192.146",
+                         u"url": u"byhlavash.chimkent.su/vdvdv.exe"}}
+                """
+        if "http" not in block.get("server", {}).get("url"):
+            url = "http://" + block["server"]["url"]
+        context = {}
+        context["date_added"] = datetime.strptime(
+            block["first_seen"], "%Y-%m-%d %H:%M:%S")
+        context["as"] = block["server"]["AS"]
+        context["country"] = block["server"]["country"]
+        context["ip"] = block["server"]["ip"]
```

Comment:
Are you 100% sure that these keys will exist? You seem to be checking for existence in L63